apps/web/src/pages/integrations/IntegrationsStorePage.tsx

```diff
@@ -151,6 +151,7 @@ export interface IIntegratedProvider {
   logoFileName: ILogoFileName;
   betaVersion: boolean;
   novu?: boolean;
+  environmentId?: string;
```

Comment:
Not your responsibility as inherited but we are exporting two exact `IIntegratedProvider` interfaces from here and `IntegrationsStoreModal`. We should share it from a type file.